metrics-proxy/src/main/resources/configdefinitions/vespa-services.def

```diff
@@ -5,6 +5,5 @@ package=ai.vespa.metricsproxy.service
 service[].configId string
 service[].name string default=""
 service[].port int default=-1
-service[].healthport int default=-1
```

Comment:
Removing fields in a config definition is not allowed.

Reply:
It should be okay as this is a new config definition introduced in this PR.

Reply:
OK, sure, then it is OK